---
timestamp: '2025-03-03T16:33:20.817132'
level: INFO
issuer: instance_solver
event_type: metadata
data:
    log_file: /home/remote/u7899572/lifted-white-plan-domain-repair/exp_logs_anu/dfs
        exhaust lp033/mprime__pprob26-err-rate-0-1.yaml
    instance_id: mprime__pprob26-err-rate-0-1
    search_algorithm: dfs
    plan_length: 6
    domain_class: mprime
    instance_name: pprob26-err-rate-0-1
    ground_plan:
    - (overcome boils entertainment haroset earth neptune)
    - (feast entertainment haroset marzipan oregon bavaria)
    - (drink rice marzipan oregon manitoba quebec moravia kentucky)
    - (feast entertainment marzipan turkey moravia kentucky)
    - (feast entertainment turkey papaya manitoba oregon)
    - (succumb boils entertainment papaya earth neptune)
    lifted_plan:
    -   - overcome
        - boils
        - entertainment
        - haroset
        - ?earth
        - neptune
    -   - feast
        - entertainment
        - haroset
        - ?marzipan
        - oregon
        - bavaria
    -   - drink
        - rice
        - marzipan
        - ?oregon
        - manitoba
        - quebec
        - ?moravia
        - kentucky
    -   - feast
        - entertainment
        - marzipan
        - turkey
        - ?moravia
        - kentucky
    -   - feast
        - entertainment
        - turkey
        - papaya
        - ?manitoba
        - oregon
    -   - succumb
        - boils
        - entertainment
        - papaya
        - earth
        - neptune


---
timestamp: '2025-03-03T16:33:20.820916'
level: INFO
issuer: instance_solver
event_type: ground_repair
data:
    repair_length: 0
    repair: ''


---
timestamp: '2025-03-03T16:43:36.715482'
level: ERROR
issuer: batch_solver
event_type: error
data:
    error_type: MemoryError
    error_message: 'Command ''[''/home/projects/u7899572/conda-envs/main/bin/python3'',
        ''instance_solver.py'', ''dfs'', ''/home/remote/u7899572/lifted-white-plan-domain-repair/input/benchmarks-G1'',
        ''/home/remote/u7899572/lifted-white-plan-domain-repair/exp_logs_anu/dfs exhaust
        lp033/mprime__pprob26-err-rate-0-1.yaml'', ''1000000'', ''mprime__pprob26-err-rate-0-1'',
        ''0.33'', ''unary'']'' died with <Signals.SIGABRT: 6>.'
    stack_trace: "Traceback (most recent call last):\n  File \"batch_solver.py\",\
        \ line 126, in worker\n    result = subprocess.run(cmd, check=True, timeout=params['timeout_seconds'],\n\
        \  File \"/home/projects/u7899572/conda-envs/main/lib/python3.8/subprocess.py\"\
        , line 516, in run\n    raise CalledProcessError(retcode, process.args,\n\
        subprocess.CalledProcessError: Command '['/home/projects/u7899572/conda-envs/main/bin/python3',\
        \ 'instance_solver.py', 'dfs', '/home/remote/u7899572/lifted-white-plan-domain-repair/input/benchmarks-G1',\
        \ '/home/remote/u7899572/lifted-white-plan-domain-repair/exp_logs_anu/dfs\
        \ exhaust lp033/mprime__pprob26-err-rate-0-1.yaml', '1000000', 'mprime__pprob26-err-rate-0-1',\
        \ '0.33', 'unary']' died with <Signals.SIGABRT: 6>.\n"
    instance_id: mprime__pprob26-err-rate-0-1
    process_info:
        cmd:
        - /home/projects/u7899572/conda-envs/main/bin/python3
        - instance_solver.py
        - dfs
        - /home/remote/u7899572/lifted-white-plan-domain-repair/input/benchmarks-G1
        - /home/remote/u7899572/lifted-white-plan-domain-repair/exp_logs_anu/dfs exhaust
            lp033/mprime__pprob26-err-rate-0-1.yaml
        - '1000000'
        - mprime__pprob26-err-rate-0-1
        - '0.33'
        - unary
        duration: '617.44'
        python_executable: /home/projects/u7899572/conda-envs/main/bin/python3
    stdout: ''
    stderr: 'terminate called after throwing an instance of ''Minisat22::OutOfMemoryException''

        '


